app/javascript/mastodon/features/home_timeline/components/critical_update_banner.tsx

```diff
@@ -0,0 +1,26 @@
+import { FormattedMessage } from 'react-intl';
+
+export const CriticalUpdateBanner = () => (
+  <div className='warning-banner'>
+    <div className='warning-banner__message'>
+      <h1>
+        <FormattedMessage
+          id='home.pending_critical_update.title'
+          defaultMessage='Critical security update available!'
+        />
+      </h1>
+      <p>
+        <FormattedMessage
+          id='home.pending_critical_update.body'
+          defaultMessage='Please update Mastodon as soon as possible!'
```

Comment:
Yes, good point, especially for the advanced column. But I think it would be a good idea to show something to explain that this banner is about *your* instance, and that it is shown because you are an admin. Maybe a badge?

And "Please update your Mastodon server as soon as possible!"?

Reply:
Changed it to “Please update your Mastodon server as soon as possible!” as suggested; not sure how to convey this is shown because you are an admin, though.

Reply:
It says "your Mastodon server", so I guess it makes it slightly more obvious?